pkg/controller/validation/account_validation_controller.go

```diff
@@ -267,15 +267,21 @@ func ValidateAwsAccountId(account awsv1alpha1.Account) error {
 	return nil
 }
 
-func (r *ValidateAccount) ValidateAccountOU(awsClient awsclient.Client, account awsv1alpha1.Account, poolOU string) error {
+func (r *ValidateAccount) ValidateAccountOU(awsClient awsclient.Client, account awsv1alpha1.Account, poolOU string, claimedAccountOU string) error {
 	// Perform all checks on the account we want.
-	inPool := IsAccountInPoolOU(account, awsClient, func(s string) bool {
-		return s == poolOU
+	correctOU := poolOU
+
+	if account.IsClaimed() {
+		correctOU = claimedAccountOU
+	}
```

Comment:
I dont think this is right, the claimed OU structure looks like this doesnt it: `base ou -> some ou based on the org that owns the cluster i think -> account`

I dont know the exact logic for how claimed accounts get grouped under the base OU, but my point is that base OU isnt the direct parent.

Reply:
Hmm, good catch. I'll have to go brush up on the OU code to double check that then.